### ISSUE TITLE
Suggestions For Improverment or I Has Broken Everything Again

### ISSUE THREAD

**Issue by Antillar** [Storkman101/Void-World#25]
I usually prefer to create a list of what I want:
1. You do not want ppl flying around the map and spoilering themselves with bosses and other stuff. So, my suggestions are: 
   a. Make sure all pumpkins are inside the city (or in the Nether under the portal, whatever). So ppl can find all of them without aimlessly looking for something in the void. Aspecially bugging is the pumpkin on the shoulder of hay man in the middle of nowhere. Why?
   b. A little addition to quest tracking system for the first boss quest (I have posted this on forum already): Add 3 command blocks, power first with EU redstone clock, output with a comparator to the other two, one displaying warning in chat, one doing the thing. Well, and fill the first one with /testfor @a[rm=300] and the second one with /tp @a x y z, like you did in feather quest in the pre-last chapter. This will tp ppl back if they are too far from the base and are about to see the structures.
   c. Why not to use End or something for boss locations? 
2. Creative energy cell, seriously? Well, i am not really sure how you were trying to save it, I just grabbed 2 autonomous activators, placed one as far as I can, used it to break the cable and to place there the other one, then I replaced the first one with redstone to break the cell. All for what? Fancy ME controller decoration?
3. Written somewhere: "do not break purple stuff", ok, ok. Well, boss locations are not really purple, so players are "allowed" to break them. But, they may not want to if there could be pretty fair way to play there. But: soldier section: they respawn. So what can be the reason to killing them if they will definately respawn and just go and knock you back again and again? Break the floor and go straight to the boss, I don't really see the other way. 
4. When you test stuff, you test stuff. When you release stuff, you release it. Please, don't mix. This is about some test NPC's, test buttons and other stuff accessible from the outside of the boss boxes.
5. Why would you ever want to complete parkour when:
   a. You can fly.
   b. There is nothing in the end.
   c. There is a guy shooting you in the butt and making you fall.
   d. There is sewage and sludge when you fall down.
   e. There is button that allows you to skip it.
   f. You can see the destination on the JourneyMap and fly straight to it.
   Also, about that guy:
   a. He is NOT unkillable. Try RM Katar.
   b. He respawns on the same place, even if there is straight hole to the void.
   c. You can make a cobble box and push him to it, saving yourself from his projectiles.
6. Why would ever bosses respawn? Isn't it lore-breaking?
7. Where exactly do I need to place the red sand? I placed one, it didn't fall, I placed another on top, then placed one more in front of it. Then I broke the two at the bottom, allowing for all three drops to fall down, that made all beacons to close. Strange... 
8. Didn't really get the idea of Qest Gate Systems everywhere near the second boss, when you can break reactor glass and kill him even without deactivating the shield, and then to suck the drop with black hole band. What would I did - summoned any mob riding on a offset minecart with a command block with several silent invisible slimes inbetween, then placing activating rails under the floor, everywhere mob can walk, activating them and using /testfor in the minecart. Then, hook up some more comblocks with regen&protection buff (count it as shield) depending on a certain stuff, and launched the thing. Hypixel did slightly less accurate version in his map on 1.5.2, why not to do that on 1.7.10? :)
9. Wiring, seriously? I thought that using transmitters/recievers for wireless stuff is a must for all mapmakers since original FTB pyramid map. Lucky you not having much redstone stuff going on, otherwise FPS would drop to the void. 
10. (good number to cut) I have rebuilt all the city. I am pretty satisfied with the city hall, reactor, brown houses, I really like the white houses in rebuilt version, but:
    a. Why would you make the greenhouse asymmetrical? One half is 1 block shorter than the other.
    b. Seared brick houses are, ehm, how to be polite, not the best build of yours. So is the hero's house being just enlarged version of those. No windows, flat roof, box-like style... well, you know :) Go have a look on FyreUK's Fallen Kingdom Set as a ruins buildig guide :D

Hour of writing, hope, this is worthy.

**Comment by Storkman101** [comment]
Sorry for taking so long to react on this one, I had holiday and everything. Also lost my github account and password but I will take some things with me. I have to look a bit more carefull to add in or not. (I am planning to make a shield around the city so you cannot fly out of a certain region and while updating the redstone I will try to make it so that you will be set to adventure mode so you cannot break stuff. But both will be added in the same patch)

**Comment by Antillar** [comment]
Nice to see you back. I have already 100% completed the map, but I would like to get new quests to do.

**Comment by Storkman101** [comment]
In the 1.1.4 update there are some new quests (not big ones sadly) in the Storage quest line. The really big ones will probably come after when I explained the autocrafting in AE a bit. I guess you know your AE shizzle already but if not now you can start a bit

**Comment by Antillar** [comment]
Well, Auto-crafting in AE2 is done in two different structures.

First structure is a Crafting CPU + Crafting Memory (the bigger size of the memory, the more complicated crafting recipes it can hold). A Crafting Monitor attached to the structure is highly recommended. I do a structure of 2x2x1 consisting of 1 Monitor, 1 CPU and 2 64k memory blocks. And hook the structure to the AE2 system.

The second structure is ME Interfaces + Molecular Assembling Chambers (MAC). ME Interface holds crafting patterns (will get to that later) and processes them in MAC it is attached to. Both MAC and ME Interface can distribute the ME system to the blocks next to it. I recommend doing a checkered pattern of MACs and ME Interfaces, where each full-size ME Interface is surrounded with at least 3 MACs and ideally 4. 

Crafting cards: first you make a crap tonn of blank cards and 1 Pattern Terminal (or smth like that) and place blank patterns in it. Then you choose Crafting Pattern and create a recipe and when you see the result, press Encode, then take the pattern and place it in the Interface surrounded by MACs. And option to craft this item will apear in any  terminal excluding ME Interface terminal and including Wireless terminal. You can select what and how much to craft, choose a CPU (if you have 2+ of them) and monitor the crafting process through crafting monitor in structure 1.

Processing cards: to automate metal dust you need any machine that makes it from ingot or ore, an ME Interface (compact one), an Import Bus and 1 Blank Pattern. Choose Processing type when encoding, specify what goes in and what should come out as registered stuff. I dont usually specify secondary grinding products ect. Then Encode. Go place it in ME Interface attached to the input side of the block (makes sence for TE stuff) and hook an Import bus to any output side. Then you can select what to craft and see the machine doing the thing.

This somehow explains AE2 autocrafting, now it is your turn to explain it to your modpack players :)

**Comment by Storkman101** [comment]
I know how it works but yeah the most difficult part is to explain it to the guys who don't know how the autocrafting works

**Comment by Storkman101** [comment]
ow btw. There is a reason why I used wiring instead of the recievers. I don't have a great experience about the wireless redstone on online servers. Every time the server restarted (with multiple servers, and multiple versions of mc) I had problems that all my recievers and transmitters lost their frequency and had to reset them. (it was still on the same frequency but it didnt work anymore). This is why I intergrated computercraft into the pack so that in the next (or the one after) I can remove a lot of wiring and make it more compact.
